src/kernels/composable_kernel/include/tensor_operation/xdlops_gemm.hpp

```diff
@@ -330,14 +382,20 @@ struct mfma_info<mfma_instr::mfma_f32_32x32x2bf16>
     static constexpr index_t cycles          = 64;
     static constexpr index_t k_base          = 2;
 
-    template <index_t MPerXdlops, index_t NPerXdlops>
-    __device__ void run(const ushort* a, const ushort* b, float* reg_c) const
+    template <index_t MPerXdlops,
+              index_t NPerXdlops,
+              index_t AStride = 1,
+              index_t BStride = 1,
```

Comment:
I thought C++ would allow default template argument only as the last arguments. I may be wrong
```
template<index_t MPerXdlops,
              index_t NPerXdlops,
              class FloatA,
              class FloatB,
              class FloatC,
              index_t AStride = 1,
              index_t BStride = 1>
```

Reply:
That's default argument to functions, not templates.

Reply:
Removed these default arguments, which is useless.